### ISSUE TITLE
Improve discoverability of Compact Folders option

### ISSUE THREAD

**Issue by mhanuszh** [microsoft/vscode#89292]
When somebody wants to disable compact folders people rarely search for terms like "compact folders", but rather for "empty folders", "flattened folders"

Discoverability of this feature could be improved if

- The Settings Search would return Compact Folders for keywords like "empty folders" and "flattened folders"
- Or it could ask on the first start

**Comment by ArturoDent** [comment]
Yes, I have seen a few recent SO questions on this - they are all duplicates - but no one knows what to search for.

**Comment by joaomoreno** [comment]
@roblourens Is there such a feature in which I can add specific keywords to setting contributions so people find them easier?

**Comment by roblourens** [comment]
Yeah, we can

- register two words as synonyms, eg "compact/flattened"
- push specific settings to the top for particular queries

if you let me know what you want, I can add those

**Comment by ArturoDent** [comment]
Definitely "flattened" should be added.  Maybe "merged"?

**Comment by roblourens** [comment]
I would still like to do these settings search tweaks, it's easy, except that there are some issues with the service the past couple weeks and I can't do it right now.

**Comment by mhanuszh** [comment]
Can this issue be expanded to be more general?

There are other situations where this would be useful, and not just in the settings search, but also in the command palette, for example: folding/unfolding could be referred as "collapse"

Also, the frequently used settings should be marked as such and elevated to the top of the list

**Comment by roblourens** [comment]
> but also in the command palette, for example: folding/unfolding could be referred to as "collapse"

We talked about a similar fuzzy search capability for the command palette awhile back, but no plans to do this in the near future.
